src/org/usfirst/frc/team4571/robot/commands/MillCommand.java

```diff
@@ -0,0 +1,48 @@
+package org.usfirst.frc.team4571.robot.commands;
+
+import org.usfirst.frc.team4571.robot.Robot;
+
+import edu.wpi.first.wpilibj.command.Command;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+
+/**
+ *
+ */
+public class MillCommand extends Command {
+
+    public MillCommand() {
+        // Use requires() here to declare subsystem dependencies
+        // eg. requires(chassis);
+    	requires(Robot.MILL_SUBSYSTEM);
+    }
+    public void forward(){
+    	Robot.MILL_SUBSYSTEM.forward(SmartDashboard.getDouble("MillForwardSpeed", 0.5));
```

Comment:
The usage of the smart dashboard, while cool, is wrong. We should also not set the speed to 0.5 here but pass it in as a constant which we can change whenever we decide what the right speed is. So create a constant in RobotConstants called MILL_CONSTANT_SPEED and reference that speed in the forward and reverse methods.